src/main/steps/respondent/task-list/content.ts

```diff
@@ -55,51 +55,93 @@ export const generateContent: TranslationFn = content => {
 const getC100Banners = userCase => {
   console.log(userCase.caseTypeOfApplication);
   const banners: Banner[] = [];
-  banners.push({
-    bannerHeading: 'Respond to an application about a child',
-    bannerContent: [
-      {
-        line1: 'Another person (the applicant) has applied to the court to make a decision about a child.',
-        line2:
-          'You should respond within 14 days of receiving the application unless the court has asked you to respond sooner.',
-      },
-    ],
-    bannerLinks: [
-      {
-        href: `${APPLICANT}${APPLICANT_CA_DA_REQUEST}`,
-        text: 'Check the application (PDF)',
-      },
-      {
-        href: RESPOND_TO_APPLICATION,
-        text: 'Respond to the application',
-      },
-    ],
-  });
+  if (userCase.orderCollection && userCase.orderCollection.length > 0) {
+    const doc = userCase.orderCollection[0];
+    const uid = doc.value.orderDocument.document_url.substring(
+      doc.value.orderDocument.document_url.lastIndexOf('/') + 1
+    );
+    console.log(userCase.state);
+    if (userCase.state !== 'ALL_FINAL_ORDERS_ISSUED') {
+      banners.push({
+        bannerHeading: 'You have a new order from the court',
+        bannerContent: [
+          {
+            line1: 'The court has made a decision about your case. The order tells you what the court has decided.',
+            line2: ' ',
+          },
+        ],
+        bannerLinks: [
+          {
+            href: `${RESPONDENT_ORDERS_FROM_THE_COURT}/${uid}`,
+            text: 'View the order (PDF)',
+          },
+        ],
+      });
+    } else {
+      banners.push({
+        bannerHeading: 'You have a final order',
+        bannerContent: [
+          {
+            line1:
+              'The court has made a final decision about your case. The order tells you what the court has decided. ',
+            line2: ' ',
+          },
+        ],
+        bannerLinks: [
+          {
+            href: `${RESPONDENT_ORDERS_FROM_THE_COURT}/${uid}`,
+            text: 'View the order (PDF)',
+          },
+        ],
+      });
+    }
+  }
   return banners;
 };
 
 const getFl401Banners = userCase => {
   console.log(userCase.caseTypeOfApplication);
   const banners: Banner[] = [];
-  banners.push({
-    bannerHeading: 'Respond to an application about a child',
-    bannerContent: [
-      {
-        line1: 'Another person (the applicant) has applied to the court to make a decision about a child.',
-        line2:
-          'You should respond within 14 days of receiving the application unless the court has asked you to respond sooner.',
-      },
-    ],
-    bannerLinks: [
-      {
-        href: APPLICANT_CA_DA_REQUEST,
-        text: 'Check the application (PDF)',
-      },
-      {
-        href: RESPOND_TO_APPLICATION,
-        text: 'Respond to the application',
-      },
-    ],
-  });
```

Comment:
Why we need to remove this? I this this will be needed as well.

Reply:
added again